config/initializers/i18n-js.rb

```diff
@@ -38,5 +38,16 @@ def self.translations
       all_translations
     end
 
+    class FallbackLocales
```

Comment:
Where is this class used? Or are you opening up an existing class and adding methods to it?